tests/test_database.py

```
@@ -1,165 +1,285 @@
+import unittest
+import os
 from datetime import datetime, timedelta
 import sqlite3
+from unittest.mock import MagicMock, patch
+from src.utils.db import PickemDB
+import psutil
```

Ruff (0.8.2)

`psutil` imported but unused

Remove unused import: `psutil`

(F401)